Scripts/build-xcframework.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
```

Comment:
When I am running the `CrashReporter XCFramework` scheme twice I get the following error:
`“CrashReporter.framework” couldn’t be copied to “ios-x86_64-maccatalyst” because an item with the same name already exists.`
I suppose it's just a clean up issue.